### PR TITLE
Enable clang builds.

### PR DESCRIPTION
**Pull request by rminnich**
Finally, at long last, we can try clang on the kernel,
once we remove these gcc options.

And, wow, it's fast.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: sys/src/9/port/devcons.c [hunk 1]
```diff
@@ -757,7 +757,7 @@ conswrite(Chan *c, void *va, int32_t n, int64_t off)
 			rebootcmd(cb->nf-1, cb->f+1);
 			break;
 		case CMpanic:
-			*(uint32_t*)0=0;
+			*(volatile uint32_t*)0=0;
```
Comment (elbing) [inline sys/src/9/port/devcons.c]: wow!

File: sys/src/9/port/initcode.c [hunk 2]
```diff
@@ -43,6 +43,8 @@ startboot(char *argv0, char **argv)
 	bind(ec, env, MAFTER);
 	bind(e, env, MCREATE|MAFTER);
 	bind(s, srv, MREPL|MCREATE);
+	write(1, "HI\n", 3);
+	write(1, boot, 10);
```
Comment (elbing) [inline sys/src/9/port/initcode.c]: Why this? init9 related?

**Comment by elbing** [comment]
Don't suffer about mouse, for me it's ok (you found buttons missing mess in struct). I wonder about init9.
"vartracking" gcc flags are not needed. I can't remember why are they there, I think when Giacomo put dwarf option too.
LGTM.

**Comment by rminnich** [comment]
Alvaro, thanks for checking this. That "HI" print was debug code I left in by mistake! NIce catch!

**Pull request merged**